capa/render/default.py

```diff
@@ -148,7 +152,60 @@ def render_attack(doc: rd.ResultDocument, ostream: StringIO):
     if rows:
         ostream.write(
             tabulate.tabulate(
-                rows, headers=[width("ATT&CK Tactic", 20), width("ATT&CK Technique", 80)], tablefmt="mixed_grid"
+                rows,
+                headers=[width("ATT&CK Tactic", 20), width("ATT&CK Technique", 80)],
+                tablefmt="mixed_grid",
+            )
+        )
+        ostream.write("\n")
+
+
+def render_maec(doc: rd.ResultDocument, ostream: StringIO):
+    """
+    example::
+
+        +--------------------------+-----------------------------------------------------------+
+        | MAEC Schema              | MAEC Value                                                |
```

Comment:
I'm not sure "Schema" fits, but can't find a clear alternative. Maybe "Object" or "Entry"?

Reply:
maybe schema is correct, do you have a reference?

Reply:
i had been struggling with the wording here too. 

maybe "MAEC analysis category"/"conclusion"?

let's keep tweaking as we find better and better wording.

Reply:
I only picked schema when I was reading the documentation. But, I agree the wording is confusing. Other candidates are: MAEC property, MAEC attribute, MAEC category or MAEC class. Let's decide on one.

Second field can be something like: MAEC result, MAEC conclusion, MAEC value.